exonum/src/helpers/fabric/details.rs

```diff
@@ -392,6 +406,47 @@ impl GenerateNodeConfig {
 
         (external_address, listen_address)
     }
+
+    /// * `key` - expects EXONUM_SERVICE_PASS or EXONUM_CONSENSUS_PASS.
+    fn get_passphrase(context: &Context, key: &str) -> Vec<u8> {
```

Comment:
I'm a bit concerned that someone may reuse `get_passphrase` somewhere later to read an existing passphase, thinking that it's a general passphrase-reading function. Currently it is geared towards this only use: reading a fresh new consensus or service password for key files. Maybe we should consider giving this function more unwieldy name like `get_new_exonum_key_file_passphrase` or something alike so that it stays an internal utility.

Reply:
It's a private method for this command.. Why would someone reuse it?
I'd rather not use such a verbose name, though I can add some comments in order to make it clear what this function does.

Reply:
@mvlabat To hold a passphrase in `Vec` it's an unsecured approach. For that propose you can use https://docs.rs/smallvec/0.6.6/smallvec/struct.SmallVec.html.

Reply:
Saw the same comment for your PR and forgot about it.. Shame! )

Reply:
I'm not sure how `SmallVec` helps there. Isn't it (slightly) insecure because we should securely zero out memory before dropping it?

Reply:
I used https://crates.io/crates/zeroize here.